### DISCUSSION TITLE
如何取得 `font-custom-4k-smaller-xxx.rar`

### DISCUSSION THREAD

**Discussion by hydai** [GpointChen/FFXIVChnTextPatch-GP#405] [General]
您好，我是提供 macOS 安裝檔案封裝的開發者。

由於有收到一些使用者希望 macOS 也有提供高畫質字體包的意見，目前打算多提供一種安裝檔案（預設是自帶高畫質字體包）。

我有注意到在 release 頁面中有提供 [font-custom-4k-smaller-20230131.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.0.1/font-custom-4k-smaller-20230131.rar)，想請問這個是可以直接從此 repository 中的資料夾取得的嗎？

如果有個固定的地方可以下載的話，用 CI 來自動包裝會容易一些。

目前提供的列表為：

```
FFXIVChnTextPatch-GP-Intel.dmg # <--- For Intel model
FFXIVChnTextPatch-GP-AppleSilicon.dmg # <--- For AppleSilicon(M1/M2/Mx)
```

預計新增：

```
FFXIVChnTextPatch-GP-4k-Intel.dmg # <--- For Intel model + 4k font
FFXIVChnTextPatch-GP-4k-AppleSilicon.dmg # <--- For AppleSilicon(M1/M2/Mx) + 4k font
```

**Comment by GpointChen** [comment]
我沒有用過自動打包，沒辦法設定固定從你貼的font-custom-4k-smaller-20230131.rar連結下載嗎？

**Reply by hydai** [reply]
我想確定這個檔案是否會跟著每次改版都變動，或者是比較長期才會變動一次？
如果是長期保持不變的話，那拿取這個連結就可以了

**Reply by GpointChen** [reply]
不會變動

**Reply by hydai** [reply]
成功使用上述的連結取得 4k font，謝謝
